src/bots/modchat/tests/messageHandler.test.js

```diff
@@ -0,0 +1,45 @@
+import client from '../../../client';
+import handlePrivateMessage from '../messageHandler';
+
+describe('handlePrivateMessage', () => {
+  test('messages when user provides no arguments', async () => {
+    client.message.content = '++mc';
+
+    await handlePrivateMessage();
+
+    expect(client.message.channel.send).toHaveBeenCalledWith(
+      'Direct message bot with ++mc -a <message> to privately message mods.\nRemove -a flag to include discord name (otherwise anonymous).'
+    );
+  });
+
+  test('messages when user does not use DMChannel', async () => {
+    client.message.content = '++mc hello world';
+
+    await handlePrivateMessage();
+
+    expect(client.message.channel.send).toHaveBeenCalledWith(
+      'Please send this through direct message to bot'
+    );
+  });
+
+  //   test('sends success message when everything is sent', async () => {
```

Comment:
this test is still WIP since there is a lot of mocking needed.

Reply:
Not sure how well this works with mocking instances so for now the test is commented out. Can probably merge in without for time being.